changelog/fix_make_double_start_end_aware_safe_navigation.md

```diff
@@ -0,0 +1 @@
+* [#407](https://github.com/rubocop/rubocop-performance/issues/407): Make `Performance/DoubleStartEndWith` aware of safe navigation. ([@earlopain][])
```

Comment:
It would probably be better to start the file name with `change_` instad of `fix_`.